coderd/database/dbauthz/dbauthz.go

```diff
@@ -785,6 +785,13 @@ func (q *querier) AcquireLock(ctx context.Context, id int64) error {
 	return q.db.AcquireLock(ctx, id)
 }
 
+func (q *querier) AcquireNotificationMessages(ctx context.Context, arg database.AcquireNotificationMessagesParams) ([]database.AcquireNotificationMessagesRow, error) {
+	if err := q.authorizeContext(ctx, policy.ActionUpdate, rbac.ResourceSystem); err != nil {
```

Comment:
cc @johnstcn to validate rbac rules

Reply:
In the other PR I added a TODO to add a notifications-specific RBAC role; I'll do that as a follow-up.